src/util/filterIncludes/index.js

```diff
@@ -12,9 +12,10 @@ import { error } from '../'
  */
 export async function filterIncludes (included, { id, type }) {
   try {
-    return included.filter(el => {
+    const filtered = included.filter(el => {
       return el.id === id && el.type === type
     })[0] || { id, type }
+    return Object.assign({}, filtered)
```

Comment:
Cloning object is mandatory to avoid meta information duplication.

Reply:
I see. Can you add a test spec for this behaviour so it doesn't get unknowingly broken in the future 👍